Makefile

```diff
@@ -29,33 +29,17 @@ OPENAPI_CODEGEN_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_CODEGEN_TAG}
 DOCKER_OPENAPI=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${OPENAPI_CODEGEN_IMAGE}
 OPENAPI_URL=https://app.swaggerhub.com/apiproxy/registry/equinix-api/fabric/4.9
 
-docker_run: clean pre-spec-patch pull docker_generate build_client
+docker_run: clean fetch pre-spec-patch pull docker_generate build_client
```

Comment:
Similar comments here as on equinix-labs/fabric-go#15.  If we move towards a standard set of high-level tasks (`fetch`, `patch`, `generate`, maybe others), it'll be easier to work across different SDK repos.